labelbox/schema/role.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+from labelbox.orm.model import Field
+from labelbox.orm.db_object import DbObject
+
+if TYPE_CHECKING:
+    from labelbox.schema.project import Project
+
+
+class Roles:
+    """
+    Object that manages org and user roles
+
+        >>> roles = client.get_roles()
+        >>> roles.valid_roles # lists all valid roles
+        >>> roles['ADMIN'] # returns the admin Role
+
+    """
+    _instance = None
```

Comment:
Is this `_instance` value suppoed to be static?

Reply:
yes. This will never change during a single user session.